api/types.go

```diff
@@ -548,6 +548,8 @@ const (
 	ShardType
 )
 
+const AllType = -1
```

Comment:
Group with PinTypes or add a comment. Golint doesn't complain?

Reply:
It did which is why I added a comment!